lua/autorun/server/sv_cfc_ammo_box_protections.lua

```diff
@@ -0,0 +1,20 @@
+local function canACFDamage( activated, targetEnt, energy, fraera, angle, inflictor, bone, gun )
+    if not IsValid( targetEnt ) then return end
+    if not IsValid( inflictor ) then return end
+
+    local targetPerson
+    if targetEnt:IsPlayer() then
+        targetPerson = targetEnt
+    else
+        targetPerson = targetEnt:CPPIGetOwner()
+    end
+
+    if not IsValid( targetPerson ) then return end
+
+    if gun:GetClass() == "acf_ammo" and targetPerson ~= inflictor then
+        return false
+    end
+end
+
+hook.Add( "ACF_BulletDamage", "CFC_PvP_CanACFDamage", canACFDamage )
```

Comment:
Will want to change this to be not pvp related, and probably say something specifically about ammo boxes (may also consider changing the method name)